CHANGELOG.md

```diff
@@ -18,7 +18,8 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 - Added missing javadocs for `:distribution:tools` modules ([#4483](https://github.com/opensearch-project/OpenSearch/pull/4483))
 - Add BWC version 2.3.1 ([#4513](https://github.com/opensearch-project/OpenSearch/pull/4513))
 - [Segment Replication] Add snapshot and restore tests for segment replication feature ([#3993](https://github.com/opensearch-project/OpenSearch/pull/3993))
-- [Security] Initial interface for native authentication support ([#4515](https://github.com/opensearch-project/OpenSearch/pull/4515))
+- [Identity] Initial interface for native authentication support ([#4515](https://github.com/opensearch-project/OpenSearch/pull/4515))
+- [Identity] Update readme for feature/identity usage
```

Comment:
Link to the issue here? Should this be `security roadmap` opposed to `readme`?

Reply:
Good call, will update the naming